Automattic/wp-calypso
# AI Logo Generator: Update events

Related to #87018

## Proposed Changes

* Changes the event names
* Adds a `context` property

## Testing Instructions



* Generate a logo and go through the possible actions like clicking feedback
* Check that they are registered and the `context` parameter is set

## Pre-merge Checklist
